jenkins.sh

```diff
@@ -42,7 +39,8 @@ virtualenv $WORKSPACE/.venv
 PYTHON=$WORKSPACE/.venv/bin/python
 PIP=$WORKSPACE/.venv/bin/pip
 (cd synapse ; $PYTHON synapse/python_dependencies.py | xargs -n1 $PIP install)
-
+$PIP install psycopg2
+$PIP install lxml
```

Comment:
We special case them in synapse for reasons. This is copied from the synapse jenkins.sh